tools/appliance/systemvmtemplate/scripts/configure_systemvm_services.sh

```diff
@@ -132,7 +132,7 @@ function configure_services() {
 
   # Disable cloud init by default
 cat <<EOF > /etc/cloud/cloud.cfg.d/cloudstack.cfg
-datasource_list: ['CloudStack']
+datasource_list: ['CloudStack', 'ConfigDrive']
```

Comment:
this can be fixed after the setup/init scripts @weizhouapache ? As cloud-init is disabled by default, the post-init or similar scripts can fix this and you won't need a new template for this?

Reply:
@rohityadavcloud 
yes, good idea.
we could use cloud-early-config to update the setting. It requires testing.
I will update this PR when #7828 is merged (new 4.18.1 systemvm template)

Reply:
PR is updated
ugly but let's see if it works